error-prone-contrib/src/main/java/tech/picnic/errorprone/refasterrules/ReactorRules.java

```diff
@@ -1761,6 +1762,51 @@ StepVerifier.FirstStep<? extends T> after(Flux<T> flux) {
     }
   }
 
+  /**
+   * Prefer {@link StepVerifier.LastStep#verify()} over a dangling {@link
+   * StepVerifier#verifyThenAssertThat()}.
+   */
+  static final class StepVerifierVerify {
+    @BeforeTemplate
+    void before(StepVerifier stepVerifier) {
+      stepVerifier.verifyThenAssertThat();
+    }
+
+    @AfterTemplate
+    void after(StepVerifier stepVerifier) {
+      stepVerifier.verify();
+    }
+  }
+
+  /**
+   * Prefer {@link StepVerifier.LastStep#verify(Duration)} over a dangling {@link
+   * StepVerifier#verifyThenAssertThat(Duration)}.
+   */
+  static final class StepVerifierVerifyDuration {
+    @BeforeTemplate
+    void before(StepVerifier stepVerifier, Duration duration) {
+      stepVerifier.verifyThenAssertThat(duration);
+    }
+
+    @AfterTemplate
+    void after(StepVerifier stepVerifier, Duration duration) {
+      stepVerifier.verify(duration);
+    }
+  }
+
+  /** Don't unnecessarily invoke {@link StepVerifier#verifyLater()} multiple times. */
+  static final class StepVerifierVerifyLater<T> {
+    @BeforeTemplate
+    StepVerifier before(StepVerifier stepVerifier) {
+      return stepVerifier.verifyLater().verifyLater();
+    }
+
+    @AfterTemplate
+    StepVerifier after(StepVerifier stepVerifier) {
+      return stepVerifier.verifyLater();
+    }
+  }
```

Comment:
The API spec mentions this is a noop. I didn't fully understand why this API may be useful for in the first place.